homework/zadanie2_pararel_accumulate.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <thread>
+#include <numeric>
+#include <algorithm>
+#include <vector>
+#include <chrono>
+#include <functional>
+
+
+template <typename IT, typename T>
+T pararelAccumulate(IT first, IT last, T init) {
+    if (!init) {
```

Comment:
tego warunku nie rozumiem. Chyba nie tak to miało być. Gdy przekazany zakres jest pusty, to wtedy zwracamy od razu init.

Reply:
Istotnie, brak jakiegokolwiek sensu, lecialem na szybkosc;) - poprawiony na:
    if (!size)
        return init;